src/pybind/mgr/orchestrator.py

```diff
@@ -696,15 +696,21 @@ class ServiceDescription(object):
     has decided the service should run.
     """
 
-    def __init__(self, nodename=None, container_id=None, service=None, service_instance=None,
+    def __init__(self, nodename=None,
+                 container_id=None, container_image=None,
+                 container_image_name=None,
+                 service=None, service_instance=None,
                  service_type=None, version=None, rados_config_location=None,
                  service_url=None, status=None, status_desc=None):
         # Node is at the same granularity as InventoryNode
         self.nodename = nodename
 
         # Not everyone runs in containers, but enough people do to
-        # justify having this field here.
-        self.container_id = container_id
+        # justify having the container_id (runtime id) and container_image
+        # (image name)
+        self.container_id = container_id                  # runtime id
+        self.container_image = container_image            # image hash
```

Comment:
Until I saw this comment, the term `conatiner_image` was a little ambiguous ...

Maybe rename instances of `container_image` -> `container_image_id` for clarity?

Reply:
:+1: 

```suggestion
        self.container_image_id = container_image_id            # image hash
```

Containers have names and ids and container images have names and ids. `container_image` is ambiguous.